website/src/components/DisplayAuthorandReadTime/styles.ts

```diff
@@ -18,13 +22,18 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
   cardActionButton: {
     textTransform: "none",
-    color: theme.palette.warning.main,
+    color: theme.palette.text.disabled,
     fontWeight: 700,
     marginRight: theme.spacing(2),
+    fontSize: '1rem',
+    padding: theme.spacing(0),
+    [theme.breakpoints.down("xs")]: {
+      fontSize: '0.875rem',
+    },
   },
   small: {
-    width: theme.spacing(3),
-    height: theme.spacing(3),
+    width: '32px',
```

Comment:
theme.spacing(4)

Reply:
theme spacing is generally used for padding and margin

Reply:
https://material-ui.com/customization/spacing/

Reply:
got it @Pallavi-PH , thanks!!